.cirrus.yml

```diff
@@ -1,21 +1,27 @@
 # $FreeBSD$
 
 freebsd_instance:
-  image: freebsd-12-1-release-amd64
+  image: freebsd-13-1-release-amd64
 
 env:
   CIRRUS_CLONE_DEPTH: 1
+  CIRRUS_CLONE_SUBMODULES: true
 
 task:
   install_script:
   - pkg install -y
     v4l_compat swig ffmpeg curl dbus fdk-aac fontconfig
     freetype2 jackit jansson luajit mbedtls pulseaudio speexdsp
-    libsysinfo libudev-devd libv4l libx264 cmake ninja
+    libpci librist libsysinfo libudev-devd libv4l libx264 cmake ninja
     mesa-libs lua52 pkgconf
+    srt
     qt5-svg qt5-qmake qt5-buildtools qt5-x11extras qt5-xml
   script:
   - mkdir build
   - cd build
-  - cmake -DUNIX_STRUCTURE=1 -GNinja ..
+  - cmake
+      -DUNIX_STRUCTURE=1
```

Comment:
`UNIX_STRUCTURE` became deprecated with the CMake Overhaul and was replaced with `LINUX_PORTABLE` (which is false by default), I think this argument could be removed in the future.

Reply:
Ah thanks - opened 7082 for that.

I also see CI/build-freebsd.sh et al added fairly recently in c993b6059a1392db5804128f30c4a793bee09769, need to take a look and understand what makes sense for FreeBSD CI now.